packages/headless/doc-parser/doc-parser.ts

```diff
@@ -40,10 +40,10 @@ const controllers: ControllerConfiguration[] = [
       ],
     },
   },
-  {
-    initializer: 'buildCategoryFacet',
-    samplePaths: {},
-  },
+  // {
+  //   initializer: 'buildCategoryFacet',
+  //   samplePaths: {},
+  // },
```

Comment:
I've commented out processing this controller until the parser is able to resolve recursive types.

Reply:
I'll uncomment it in my PR.